ee/main.go

```diff
@@ -137,6 +137,8 @@ func main() {
 		NodeSelector:                   nodeSelector,
 		Tolerations:                    tolerationsSlice,
 		Affinity:                       affinity,
+		PhfsEnabled:                    viper.GetString("jobSubmission.phfsEnabled") == "true",
```

Comment:
A little confused that the PhfsEnabled is string type. Could we make it boolean type?